src/main/webapp/css/srp_style.css

```diff
@@ -22,6 +22,19 @@
   transition: box-shadow .25s;
   border-radius: 2px;
   width: 400px;
+  height: 260px;
+  overflow: scroll;
+}
+
+.card::-webkit-scrollbar {
```

Comment:
Just a suggestion to potentially consider but do we maybe want the hyperlinked title to stay fixed at the top of the card for easy clickability while the rest of the description/code section of the card is the scrollable only section?

I also think it seems fine this current way too though.

Reply:
I like the idea of a fixed title! @irisliu77 thoughts?

Reply:
yea, i think it's better

Reply:
Alright, I'll open an issue for this to be done in a separate PR